src/rust/src/bridge.rs

```diff
@@ -166,7 +166,7 @@ pub(crate) mod rust_bridge {
             instruction_limit: u32,
             hf_buf: &CxxBuf,
             resources: CxxBuf,
-            resources_ext: CxxBuf,
+            resources_ext: &Vec<u32>,
```

Comment:
Nit: Should this be updated to `restored_rw_entry_indices`?

Reply:
Fixed